ydb/core/fq/libs/checkpoint_storage/storage_proxy.cpp

```diff
@@ -201,19 +226,27 @@ void TStorageProxy::Bootstrap() {
         const auto& gcConfig = Config.GetCheckpointGarbageConfig();
         ActorGC = Register(NewGC(gcConfig, CheckpointStorage, StateStorage).release());
     }
-    Initialize();
+
+    Send(NKikimr::NConsole::MakeConfigsDispatcherID(SelfId().NodeId()),
+        new NKikimr::NConsole::TEvConfigsDispatcher::TEvSetConfigSubscriptionRequest({NKikimrConsole::TConfigItem::FeatureFlagsItem}));
+
     Become(&TStorageProxy::StateFunc);
+    FeatureFlags = NKikimr::AppData()->FeatureFlags;
 
     LOG_STREAMS_STORAGE_SERVICE_INFO("Successfully bootstrapped TStorageProxy " << SelfId() << " with connection to "
         << StorageConfig.GetEndpoint().data()
         << ":" << StorageConfig.GetDatabase().data())
 }
 
-void TStorageProxy::Initialize() {
-    LOG_STREAMS_STORAGE_SERVICE_TRACE("Initialize");
+void TStorageProxy::StartInitialization() {
+    LOG_STREAMS_STORAGE_SERVICE_ERROR("StartInitialization, EnableSecureScriptExecutions: " << FeatureFlags.GetEnableSecureScriptExecutions());
```

Comment:
Тут наверное стоит выставить приоритет логов в `INFO`:

```suggestion
    LOG_STREAMS_STORAGE_SERVICE_INFO("StartInitialization, EnableSecureScriptExecutions: " << FeatureFlags.GetEnableSecureScriptExecutions());
```